src/ElevatorBooking.tsx

```diff
@@ -410,7 +353,17 @@ export default function ElevatorBooking(): JSX.Element {
           <div className="section flex-grow">
             <Grid container spacing={5}>
               <Grid item xs={12}>
-                <h4 className="center">Reserve an Amenity</h4>
+                <h4 className="center">Reserve Elevator</h4>
+                <Alert severity="info">
+                  <AlertTitle>Available hours</AlertTitle>
```

Comment:
It seems that on mobile, the font size is much bigger than the other alerts from other pages, is this desired?

Reply:
Oh weird. Not intentional. Did you want to also debug this one? Only if you have time, I'm having to focus on something else for most of today.

Reply:
Sure, I can try adding changes for both!

Reply:
I have some PRs out for the checkbox error checking, however I'm having trouble debugging the font problems. I was using chrome's mobile simulator, but it seems that on another ios simulator, some of the alerts on the page no longer have this problem.

Reply:
Let's not worry about the font problem for now. Our bigger issue is how bad it is in general on mobile.